tests/resources/inputs/normalization/NormalizationRecord.json

```diff
@@ -0,0 +1,137 @@
+                0.05
+              ]
+            },
+            "particleBounds": {
+              "wavelength": {
+                "minimum": 0.7,
+                "maximum": 3.5
+              },
+              "tof": {
+                "minimum": 2742.6,
+                "maximum": 13713
+              }
+            },
+            "pixelGroup":{
+              "pixelGroupingParameters": [
+               {
+                 "groupID": 1,
+                 "twoTheta": 2.1108948177427838,
+                 "dResolution": {
+                   "minimum": 0.40224298673776404,
+                   "maximum": 2.0112149336888203
+                 },
+                 "dRelativeResolution": 0.002771822370612031
+               },
+               {
+                 "groupID": 2,
+                 "twoTheta": 1.82310673131693,
+                 "dResolution": {
+                   "minimum": 0.44278229953817677,
+                   "maximum": 2.213911497690884
+                 },
+                 "dRelativeResolution": 0.003234525153614593
+               },
+               {
+                 "groupID": 3,
+                 "twoTheta": 1.5352228379083572,
+                 "dResolution": {
+                   "minimum": 0.5040188478793922,
+                   "maximum": 2.5200942393969608
+                 },
+                 "dRelativeResolution": 0.004014748859873182
+               },
+               {
+                 "groupID": 4,
+                 "twoTheta": 1.440276389170165,
+                 "dResolution": {
+                   "minimum": 0.530714277849788,
+                   "maximum": 2.6535713892489396
+                 },
+                 "dRelativeResolution": 0.004309856397320412
+               },
+               {
+                 "groupID": 5,
+                 "twoTheta": 1.1543988461042238,
+                 "dResolution": {
+                   "minimum": 0.6414024711539658,
+                   "maximum": 3.207012355769829
+                 },
+                 "dRelativeResolution": 0.005408489642872038
+               },
+               {
+                 "groupID": 6,
+                 "twoTheta": 0.8690010092470938,
+                 "dResolution": {
+                   "minimum": 0.831438096217031,
+                   "maximum": 4.157190481085156
+                 },
+                 "dRelativeResolution": 0.007548513084996265
+               }
+             ]
+            },
+            "defaultGroupingSliceValue": 5,
+            "fwhmMultiplierLimit": {
+              "minimum": 1.3,
+              "maximum": 1.3
+            }
+          },
+          "seedRun": 57514,
+          "creationDate": "2023-06-05T15:49:59.583421",
+          "name": "test",
+          "version": 0
+    },
+    "workspaceNames": [
+        "fitted_strippedFocussedData_stripped_0",
+        "fitted_strippedFocussedData_stripped_1",
+        "fitted_strippedFocussedData_stripped_2",
+        "57514_calibration_reduction_result"
```

Comment:
Is it necessary to include the workspace names in the record?  They will be deleted when SNAPRed closes.